helm/cert-manager-app/templates/controller-deployment.yaml

```
@@ -23,6 +23,9 @@ spec:
         prometheus.io/scrape: 'true'
         prometheus.io/port: '9402'
       {{- end }}
+      {{- if .Values.controller.aws.role }}
+      iam.amazonaws.com/role: {{ .Values.controller.aws.role }}
```

```suggestion
        iam.amazonaws.com/role: {{ .Values.controller.aws.role }}
```
Indentation looks wrong